analyses/survival-analysis/README.md

```diff
@@ -0,0 +1,76 @@
+## Perform survival analysis for samples with 
+
+**Module authors:** Candace Savonen (ALSF CCDL), Run Jin (D3b), Stephanie J. Spielman (ALSF CCDL), Jo Lynne Rokita (D3b)
+
+In this module, we perform survival analyses by Kaplan-Meier, log-rank, and/or cox regression for samples in OpenPBTA using different covariates. 
+Detailed information about covariates can be found below.
+
+
+### Usage
+```sh
+bash run-survival.sh
+```
+
+### Module contents
+
+#### Function
+`util/survival_models.R` performs kaplan-meier, log rank, or cox regression for univariate and/or multivariate analyses.
+- The `survival_analysis` function performs survival modeling and returns a list with three objects:
+    1. The original model fit object
+    2. The summary table
+    3. The original dataframe
+- The `fit_save_model` function wraps the function `survival_analysis()` to perform and export a survival model
+
+
+#### Scripts
+
+`survival-analysis_template.Rmd` shows the usage of function `survival_analysis` to perform Kaplan-Meier, log-rank and cox regression survival analyses. 
+Particularly, it demonstrates runninng Kaplan-Meier and log-rank survival analysis on categorial variable, using `germline_sex_estimate` as an example.
+It also demonstrates how to run cox regression survival analysis on continuous variable, using `tmb` (tumor mutation burden) as an example.
+
+`survival-analysis_subtypes.Rmd` runs the following analyses:
+1. Univariate models
+- Kaplan-Meier survival analysis on HGG samples (minus oligodendrogliomas) by `molecular_subtype`
+- Cox regression univariate analysis on HGG samples (minus oligodendrogliomas) by `molecular_subtype`
+
+`survival-analysis_tp53_telomerase.Rmd` runs the following analysis:
+1. Univariate models 
```

Comment:
Parentheticals are giving different types of information here. I think it's fine to just include the type of test, and more details about those variables are in the notebooks themselves.